lib/frizz/remote.rb

```diff
@@ -14,13 +14,26 @@ def files
     end
 
     def upload(file, key, options = {})
+      encoding_option = {}
+
+      if options[:prefer_gzip]
+        return unless key.ends_with? '.gz'
```

Comment:
I made a mistake in the logic here. The intention was to upload all files like normal, unless it's a gzipped file and the prefer_gzip option is true. In that case, specify `content-encoding` as `gzip` and remove the `.gz` file extension for the upload options.

The `Ignorance` class takes care of ignoring files that have a gzip version available, and will fall back on the regular file if it doesn't exist (assuming the prefer_gzip option is true).